packages/cms-lib/lib/walk.js

```diff
@@ -1,22 +1,31 @@
 const fs = require('fs');
 const path = require('path');
+const { logger } = require('../logger');
 
 function walk(dir) {
   const processFiles = files => {
     return Promise.all(
       files.map(file => {
         return new Promise((resolve, reject) => {
           const filepath = path.join(dir, file);
-          fs.stat(filepath, (error, stats) => {
+          fs.lstat(filepath, (error, stats) => {
```

Comment:
FYI going forward, [`fs-extra`](https://www.npmjs.com/package/fs-extra) is already a project dep.  It promisifies most functions from `fs` and adds a some convenience functions.

Reply:
I switched over to using `fs-extra` and making everything sync, but then rolled back due to potential performance issues.

We are walking many files(potentially) and blocking the thread for each one could really slow things down. I think using `fs-extra` is a good idea going forward, but for now it's a simpler change without it.